packages/inter-protocol/test/vaultFactory/test-vaultFactory.js

```diff
@@ -257,13 +234,11 @@ const getRunFromFaucet = async (t, amount) => {
 };
 
 /**
- * @deprecated use the subscriber directly
- *
  * Vault offer result used to include `publicNotifiers` but now is `publicSubscribers`.
  *
  * @param {UserSeat<VaultKit>} vaultSeat
  */
-const legacyOfferResult = vaultSeat => {
+export const legacyOfferResult = vaultSeat => {
```

Comment:
tests must not export. when you import from a test file, Ava includes all its tests in the run.

Reply:
I didn't see your comment here until I'd noticed the problem and pulled out the shared helper functions because it was running tests I hadn't asked for. TIL that it's intentional.